server/src/main/java/org/candlepin/controller/Entitler.java

```diff
@@ -199,7 +199,8 @@ public void adjustEntitlementQuantity(Consumer consumer, Entitlement ent,
         Consumer consumer = data.getConsumer();
         Owner owner = data.getOwner();
 
-        if ((!consumer.isDev() && owner.isAutobindDisabled()) || owner.isContentAccessEnabled()) {
+        if ((!consumer.isDev() && owner.isAutobindDisabled()) &&
+            owner.isAutobindHyperVisorDisabled() || owner.isContentAccessEnabled()) {
```

Comment:
This will catch all binds, not just those for hypervisors. We'll need to be more thorough in this check.

Reply:
Any suggestions to make it better?